sequel-ace.xcodeproj/project.pbxproj

```diff
@@ -2227,6 +2229,7 @@
 		584756F7120A1B0B0057631F /* QuickLook Plugin */ = {
 			isa = PBXGroup;
 			children = (
+				964908C4249A77CF0052FC4A /* ssh_config */,
```

Comment:
On the filesystem, `ssh_config` is in the right place: `Resources`, but in Xcode, it's in the `QuickLook Plugin` group, just a little nitpick.

Reply:
Good catch, will fix tonight

Reply:
Think I fixed it! https://github.com/Sequel-Ace/Sequel-Ace/pull/28/commits/25dac4badfc621c8f93febaaddde12a84681c205

Reply:
Yep, looks good.